### DISCUSSION TITLE
协助编辑权限申请（存档）

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#63] [公告]
### 这个帖子在这里仅供留档了。
   
  　
  　
  　 
---

我先看看现在这个人数够不够管理，如果人手不够我再继续加……大家可以继续先回着，只是暂时不会来一个放一个 =。=……
通过了的回复我会直接删掉。

---

如果感兴趣想帮帮忙的话，可以回复这个帖子来申请加权限！要求嘛……**你的 GitHub 账号至少应该有些活跃度，有 MC 相关存储库的贡献就更好啦。**

这一切都不是强制的，没有 “任务要求”，有空了就看看，没空不管也完全无所谓，放轻松……就是说，如果你看到有憨憨反馈但又只能干看着很恼火的话，就顺手来申请一下权限都可以。

下面是你能帮上忙的事 =。=！

---

### Issues

所有的新反馈都自动带有 `新反馈` 标签。查看这些反馈，**先移除** `新反馈` **标签** ，再根据情况：
- **和已有反馈重复：** 回复重复反馈的编号，并添加 `重复` 标签
- **与 PCL2 无关：** 例如 MC 进不去服务器，不会安装 Mod 等。回复建议其在 [提问板块](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E6%8F%90%E9%97%AE) 发帖，并添加 `忽略` 标签
- **不知道在说啥或是纯骂人的：** 添加 `忽略` 标签，如果是纯骂人的或很恶劣的可以添加 `删除` 标签
- **缺少信息：** 例如崩溃未上传错误报告，Bug 未上传日志等，回复具体情况并添加 `需要补充信息` 标签
- **正常：** 添加 `待处理` 标签，代表已通过初步确认，开发者就会继续处理（如果是 Bug 反馈的话最好确认一下可以复现）

此外，也可以看看 `需要社区帮助` 和 `需要补充信息` 的反馈，看看有没有帮的上忙的……

这三种标签（ `新反馈` 、 `需要社区帮助` 和 `需要补充信息` ）颜色都是橘红色的，也就是说看看列表里带橘红色标签的就行啦。点击 [这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aopen+is%3Aissue+label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E9%9C%80%E8%A6%81%E8%A1%A5%E5%85%85%E4%BF%A1%E6%81%AF+) 也可以快速查看筛选了这三种标签的反馈列表。

**注意：** 你只可以也只需要按上述说明修改标签，不要进行任何其他操作。机器人会自动根据标签进行下一步处理。

---

### Discussions

看一看 [提问板块](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E6%8F%90%E9%97%AE) 有没有能回复的。

如果里面有恶意辱骂等不正常帖子，可以锁帖或者删除。

---

哦对了，上述的事情并不能兑换活跃橙隐藏主题 —— 如果能兑换的话，就会有很多并不打算认真帮忙的人来瞎混，反而会让社区的管理质量变差。

多谢大家啦 Orz！

**Comment by z0z0r4** [comment]
回复.jpg，有mc相关仓库，来申请

**Comment by Renna42** [comment]
回复.psd

**Reply by Renna42** [reply]
有mc相关仓库

**Reply by Renna42** [reply]
有时间清理些比较zz的反馈

**Comment by LTCatt** [comment]


**Reply by z0z0r4** [reply]
哦嚯，来晚了可还行.........

**Comment by alanail123** [comment]
可以帮帮忙

**Comment by dolphinmiao** [comment]
我也来帮！w

**Comment by LTCatt** [comment]
由于 @bwtx2022 存在恶意利用权限倾向，并已因此提交了 Pull Request #91 ，已永久撤销此人的编辑权限。
![image](https://user-images.githubusercontent.com/40348278/182997589-824b6bc9-b529-4664-bc9b-c139ef185052.png)
@beiyaohhhc 同时表现出了参与倾向，其相关权限申请已拒绝。
![image](https://user-images.githubusercontent.com/40348278/182997547-8c8e39f3-463c-4348-a022-af58b00012c3.png)

**Reply by Pigeon0v0** [reply]
> 由于 @bwtx2022 存在恶意利用权限倾向，并已因此提交了 #91 ，已永久撤销此人的编辑权限。
> @beiyaohhhc 同时表现出了参与倾向，其相关权限申请已拒绝。

我在隔壁说句话不至于吧...

**Reply by LTCatt** [reply]
你不止说句话，你 PR 已经交了

**Reply by Pigeon0v0** [reply]
> 你不止说句话，你 PR 已经交了

这个本意不是瞎搞（

**Reply by LTCatt** [reply]
你 PR 里但凡没有那个 V ME 50 和 WE ARE RICH 我也不至于认为你在瞎搞.jpg

**Reply by Pigeon0v0** [reply]
> 你 PR 里但凡没有那个 V ME 50 和 WE ARE RICH 我也不至于认为你在瞎搞.jpg

我对仓库也没有读写权限，添加的徽章也只是玩梗，毕竟 PCL 也是一个比较有活力的启动器（

我也没对 Issues 列表有任何不当操作

**Reply by Lavoltz** [reply]
不太看得懂这个处理逻辑。。这个徽章有什么问题吗

**Comment by LTDove** [comment]
申请.jpg

**Comment by bigbird2007** [comment]
申请.png

**Comment by YangHaoNing-CN** [comment]
申请.gif

**Comment by bluequestions** [comment]
申请.pptx

**Comment by WTP016-CN** [comment]
怎么各种乱七八糟的格式越来越多了 草.png

**Reply by Big-Cake-jpg** [reply]
万物皆可扩展名.webp（

**Reply by LTDove** [reply]
确实.word

**Reply by Jordan8315** [reply]
确实.dat

**Reply by zkitefly** [reply]
确实.sh

**Reply by ghost** [reply]
可还行.cat

**Reply by Nightingale0504** [reply]
确实.gfe

**Reply by dolphinmiao** [reply]
确实.jpg

**Comment by EikoocS** [comment]
申请.json

**Comment by xiaoyucshijie** [comment]
**申请.png**

**Comment by ghost** [comment]
我我我，我也想要.exe

**Comment by Jordan8315** [comment]
我也申请.gsp

**Reply by Nightingale0504** [reply]
几何画板项目文件是吧（让我看看做的什么？

**Reply by Jordan8315** [reply]
:-(

**Comment by xxtg666** [comment]
建议给一个重命名issue的权限……有些issue的标题写的不是很清楚

**Comment by Xiaobei08** [comment]
申请.jar

**Comment by thehrz** [comment]
申请.cpp

**Comment by WTP016-CN** [comment]
建议就是搞个考核（
太多人来申请了（）

**Reply by LTCatt** [reply]
现在人看起来还够，暂时还不用加人 =。=

**Comment by BlastFireL** [comment]
申请.bmp

**Comment by ZanderJi** [comment]
申请.cmd

**Comment by Stfrostec** [comment]
摸鱼申请。orz

**Comment by MeowStarrySea** [comment]
申请.json

**Comment by zkitefly** [comment]
想要个申请.tga

**Comment by WForst-Breeze** [comment]
申请.jar.disabled

**Comment by G-mare** [comment]
申请.sh

**Comment by LanYangYang-114** [comment]
凑热闹.psd，主要是看憨憨反馈不爽很久了（）

**Comment by wjy0411** [comment]
可以提建议
1.加quilt端的下载
2.加planetminecraft 或是curse forge 资源包下载

**Reply by mcwuguodadi** [reply]
申请协助编辑权限的地方不是用来提建议的...

**Comment by liubanlaobanzhang** [comment]
报名.dat

（PS：学业问题不会经常来看 但能看到的时候一定帮忙）

> 奇奇怪怪的拓展名增加了

**Comment by Nightingale0504** [comment]
申请.manifest

**Comment by HRxiaohu** [comment]
申请.cs

**Comment by ghost** [comment]
我想要.vb

**Comment by JunketPear** [comment]
申请.dll

**Comment by Super-XiaoNian** [comment]
申请.bat

**Comment by tripleInt** [comment]
申请.cpp

**Comment by z0z0r4** [comment]
各位别申请了...后面来的 No one successed
~~申请成功的不准备把这个 close 了吗，明明都不要人了~~

**Comment by ZerkyLiu** [comment]
下次申请的时候提醒我呀（没申请上）

**Comment by ZerkyLiu** [comment]
申请.bat（下次的申请）

**Comment by YB-10032** [comment]
申请.hap

**Comment by HaiHomeQwQ** [comment]
申请.prproj
